src/System.Security.Cryptography.Xml/tests/EndToEnd.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml;
+using Xunit;
+
+namespace System.Security.Cryptography.Xml.Tests
+{
+    public class EndToEnd
```

Comment:
@krwq Could we create a folder for MSDN samples and move the MSDN samples into there? That keeps them separate and focused on education instead of breadth and edge cases.